### PR TITLE
[WIP] - Training slim

### PR DESCRIPTION
**Pull request by hua-phuoc-mulodo**
[training-slim] - Add template slim

### CODE REVIEW

File: app/views/devise/sessions/new.html.slim [hunk 1]
```diff
@@ -0,0 +1,19 @@
+h2 Log in
+
+= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
+  div class="form-group"
```
Comment (tran-hai-mulodo) [inline app/views/devise/sessions/new.html.slim]: @hua-phuoc-mulodo 
We should use slim shortcuts for tag attributes.
e.g: 
`.form-group`
Read here for more detail: [Slim shortcuts](https://github.com/slim-template/slim#shortcuts)

File: app/views/posts/_form.html.slim [hunk 2]
```diff
@@ -0,0 +1,30 @@
+= form_for(@post, :remote => true, :html => { :multipart => true }) do |form|
+  div id="error_explanation"
```
Comment (tran-hai-mulodo) [inline app/views/posts/_form.html.slim]: @hua-phuoc-mulodo 
We should use slim shortcuts for tag attributes.
e.g: 
`#error_explanation`
Read here for more detail: [Slim shortcuts](https://github.com/slim-template/slim#shortcuts)

File: app/views/posts/index.html.slim [hunk 3]
```diff
@@ -0,0 +1,27 @@
+h1 Post
+= link_to('New Post', new_post_path)
+table class="table table-striped"
```
Comment (tran-hai-mulodo) [inline app/views/posts/index.html.slim]: @hua-phuoc-mulodo 
We should use slim shortcuts for tag attributes.
e.g: 
`table.table.table-striped`
Read here for more detail: [Slim shortcuts](https://github.com/slim-template/slim#shortcuts)

File: app/views/posts/show.html.slim [hunk 4]
```diff
@@ -0,0 +1,18 @@
+p id="notice" = notice
```
Comment (tran-hai-mulodo) [inline app/views/posts/show.html.slim]: @hua-phuoc-mulodo 
We should use slim shortcuts for tag attributes.
e.g: 
```
p#notice
  = notice
```
Read here for more detail: [Slim shortcuts](https://github.com/slim-template/slim#shortcuts)

**Comment by tran-hai-mulodo** [comment]
@hua-phuoc-mulodo 
BTW, please remove unuse .html.erb files as below:

```
app/views/devise/registrations/edit.html.erb
app/views/devise/registrations/new.html.erb
app/views/devise/sessions/new.html.erb
app/views/pages/welcome.html.erb
app/views/posts/_form.html.erb
app/views/posts/edit.html.erb
app/views/posts/index.html.erb
app/views/posts/new.html.erb
app/views/posts/show.html.erb
```
Thanks.

**Comment by hua-phuoc-mulodo** [comment]
I fixed all your reviews. Please review it, thanks.

**Comment by tran-hai-mulodo** [comment]
@hua-phuoc-mulodo 
I checked. It's OK. Thanks.

**Pull request merged**